packages/front-end/components/Experiment/ExperimentVariationsWrapper.tsx

```diff
@@ -0,0 +1,115 @@
+import { Variation } from "back-end/types/experiment";
+import {
+  closestCenter,
+  DndContext,
+  KeyboardSensor,
+  PointerSensor,
+  useSensor,
+  useSensors,
+} from "@dnd-kit/core";
+import {
+  arrayMove,
+  rectSortingStrategy,
+  SortableContext,
+  sortableKeyboardCoordinates,
+} from "@dnd-kit/sortable";
+import { generateVariationId } from "@/services/features";
+import { GBAddCircle } from "../Icons";
+import { SortableExperimentVariationCard } from "./SortableVariationData";
+
+export type SortableExperimentVariation = Variation & {
+  id: string;
+};
+
+export interface Props {
+  variations: SortableExperimentVariation[];
+  setVariations?: (variations: SortableExperimentVariation[]) => void;
+  className?: string;
+}
+
+export default function ExperimentVariationsWrapper({
+  variations,
+  setVariations,
+  className = "",
+}: Props) {
+  const sensors = useSensors(
+    useSensor(PointerSensor),
+    useSensor(KeyboardSensor, {
+      coordinateGetter: sortableKeyboardCoordinates,
+    })
+  );
+
+  function getVariationIndex(id: string) {
+    for (let i = 0; i < variations.length; i++) {
+      if (variations[i].id === id) return i;
+    }
+    return -1;
+  }
+
+  return (
+    <div className={className}>
+      <label>Variations</label>
+      <div className="row">
+        <DndContext
+          sensors={sensors}
+          collisionDetection={closestCenter}
+          onDragEnd={({ active, over }) => {
+            if (active.id !== over.id) {
+              const oldIndex = getVariationIndex(active.id);
+              const newIndex = getVariationIndex(over.id);
+
+              if (oldIndex === -1 || newIndex === -1) return;
+
+              const newVariations = arrayMove(variations, oldIndex, newIndex);
+
+              setVariations(newVariations);
+            }
+          }}
+        >
+          <SortableContext items={variations} strategy={rectSortingStrategy}>
+            {variations.map((variation, i) => (
+              <SortableExperimentVariationCard
+                i={i}
+                variation={variation}
+                variations={variations}
+                setVariations={setVariations}
+                key={variation.id}
+              />
+            ))}
+          </SortableContext>
+        </DndContext>
+        <div
+          className="col-lg-6 col-md-6 mb-2 text-center"
+          style={{ minWidth: 200 }}
+        >
+          <div
+            className="p-3 h-100 d-flex align-items-center justify-content-center"
+            style={{ border: "1px dashed #C2C5D6", borderRadius: "3px" }}
```

Comment:
Please use a dark mode-safe colour. See the theme variables and other CSS for how to use it, e.g. `1px dashed var(--some-color);`